src/app/models/response.ts

```diff
@@ -1,12 +1,30 @@
 /* eslint-disable max-classes-per-file */
-export class Response {
+
+import { HttpTableData } from './http-table-data';
+
+export class Re {
```

Comment:
Was this name change required?

Reply:
Yes, Response is a reserved class name is Typescript.

Reply:
This is deprecated code and has been removed.